Incorrect file mode in lmdb_cat

Since I was looking at `go vet` output, I figured that I should also try `staticcheck`, which is useful in catching all kinds of errors. Mostly problems were in test code, but it did spot one issue in non-test code:

```
exp/cmd/lmdb_cat/main.go:181:43: file mode '644' evaluates to 01204; did you mean '0644'? (SA9002)
```

This seems to be an actual problem. PR incoming.

--- a/exp/cmd/lmdb_cat/main.go
+++ b/exp/cmd/lmdb_cat/main.go
@@ -178,7 +178,7 @@ func cat(path string, opt *catOptions) error {
 	if err != nil {
 		return err
 	}
-	err = env.Open(path, lmdbcmd.OpenFlag(), 644)
+	err = env.Open(path, lmdbcmd.OpenFlag(), 0644)
 	defer env.Close()
 	if err != nil {
 		return err

